schema/compose-spec.json

```diff
@@ -1522,6 +1562,32 @@
       "patternProperties": {"^x-": {}}
     },
 
+    "model": {
+      "type": "object",
+      "description": "AI Model for the Compose application.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Custom name for this model."
+        },
+        "model": {
+          "type": "string",
+          "description": "AI Model to run."
+        },
+        "context_size": {
+          "type": "integer"
+        },
+        "runtime_flags": {
+          "type": "array",
+          "items": {"type": "string"},
+          "description": "Raw runtime flags to pass to the inference engine."
+        }
```

Comment:
Should we propose also `endpoint_var` as attribut for main model definition, so it will allow user to define custom env variable for the model endpoint once for all the services and let them override it per service, wdyt?